### DISCUSSION TITLE
Need help with installing pandas

### DISCUSSION THREAD

**Discussion by LavinaPanjwani** [microsoft/pylance-release#2225] [Q&A]
Hi,
I have just started learning python. And i wa trying to install pandas using following command:
python3 pip install pandas

It installed panda package but still following error is showing up. 
ModuleNotFoundError: No module named 'pandas'

I'm using Windows 11 Home; (64 bits) and IDE is VS Code with extensions - Python and Pylance.
can anyone guide me with my issue?

**Comment by bschnurr** [comment]
Checkout
https://code.visualstudio.com/docs/python/environments#_create-a-virtual-environment
https://code.visualstudio.com/docs/python/environments#_select-and-activate-an-environment

**Comment by gramster** [comment]
It might help to select your Python environment from within VS Code, then open the terminal in VS Code, and run pip install from within that terminal. See https://code.visualstudio.com/docs/python/python-tutorial
